app/models/presenter/auction.rb

```diff
@@ -32,6 +32,7 @@ def initialize(auction)
       :to_param,
       :type,
       :updated_at,
+      :lowest_bid,
```

Comment:
just trying to wrap my head around this ... where is `lowest_bid` used in this PR where it wasn't used before?